### PR TITLE
Homework 2

### PR DESCRIPTION
**Pull request by Babashonok**

### CODE REVIEW

File: Equation.java [hunk 1]
```diff
@@ -6,8 +6,8 @@
 public class Equation {
```
Comment (AlexanderSoroka) [inline Equation.java]: Missed class comment

File: Equation.java [hunk 2]
```diff
@@ -6,8 +6,8 @@
 public class Equation {
 
    /** 
-   *check possibility of the "String -> Double" conversion
-   */
+    *check possibility of the "String -> Double" conversion
```
Comment (AlexanderSoroka) [inline Equation.java]: Missed space in comment between * and first word

File: Equation.java [hunk 3]
```diff
@@ -6,8 +6,8 @@
 public class Equation {
 
    /** 
-   *check possibility of the "String -> Double" conversion
-   */
+    *check possibility of the "String -> Double" conversion
+    */
    public static boolean checkString(String string) {
```
Comment (AlexanderSoroka) [inline Equation.java]: Missed argument description in comment

**Comment by Babashonok** [comment]
bad pull request

**Pull request closed without merging**